runtime/compiler/optimizer/JProfilingValue.cpp

```diff
@@ -476,22 +482,22 @@ TR_JProfilingValue::lowerCalls()
  *                         |                                                           |
  *                         |--------------------------------|                          |
  *                         |                                |                          |
- *                         v                                |                          |
- *          |------------------------------|                |                          |
- *          | quickInc                     |                |                          |
- *          |------------------------------|                |                          |
- *          | istorei                      |                |                          |
- *          |    al/aiadd                  |                v                          |
- *          |       aconst <table address> |        |-------------------------------|  |
- *          |       l/imul                 |        | helper                        |  |
- *          |           l/iselect          |        |-------------------------------|  |
- *          |          width               |        | call TR_jProfile32/64BitValue |  |
- *          |     iadd                     |        |    value                      |  |
- *          |        iloadi                |        |    table address              |  |
- *          |           => al/aiadd        |        |-------------------------------|  |
- *          |        iconst 1              |                         |                 |
```

Comment:
I accidentally removed the `iconst 1` from this diagram.  I will fix that along with addressing other review comments.

Reply:
Fixed in commit 85223c14b211c916c29f333009a0b8efce7a00f7